contacts/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+
+class NewContactForm(forms.Form):
+    first_name = forms.CharField(label='Enter firstname...', max_length=30, required=True)
+    last_name = forms.CharField(label='Enter lastname...',  max_length=30, required=True)
+    note = forms.CharField(label='Enter note...', max_length=30, required=True)
```

Comment:
Please increase the `max_length` of note field. Also use TextField to render this.